InterviewFlashCards/QueryManager.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  QueryManager.swift
+//  InterviewFlashCards
+//
+//  Created by Charles Kang on 11/2/16.
+//  Copyright © 2016 Varindra Hart. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+enum RequestType: Int {
+    case RequestTypeiOS = 1
```

Comment:
```    func setSectionTypeForViewController(vc: QuestionsViewController, withValue value: String) {
        if let sectionValue = SectionQuestionType(rawValue: value) {
            vc.section = sectionValue
            print(vc.section)
        }
    }``` App never hits inside the `if let`, both enums are of type String too. Not sure what you mean by map to the string endpoint!